### ISSUE TITLE
`featuretools.list_primitives()` returning wrong descriptions

### ISSUE THREAD

**Issue by csala** [Featuretools/featuretools#390]
### Bug Description

In featuretools version 0.6.0, the method `featuretools.list_primitives()` returns the same description for all the primitives of the same type. Also, the description stays consistent if the method is called multiple times, but changes every time the intepreter is restarted:

```
In [1]: import featuretools                                                                                                                                                                                                                                                    

In [2]: featuretools.list_primitives().head()                                                                                                                                                                                                                                  
Out[2]: 
           name         type                     description
0  percent_true  aggregation  Test if all values are 'True'.
1           any  aggregation  Test if all values are 'True'.
2      num_true  aggregation  Test if all values are 'True'.
3          last  aggregation  Test if all values are 'True'.
4          mode  aggregation  Test if all values are 'True'.
```

**Comment by gsheni** [comment]
@csala we are aware of this bug and have a fix coming soon

**Comment by gsheni** [comment]
Fixed #391
